src/isa/riscv64/instr/fp.c

```diff
@@ -40,6 +40,9 @@ void fp_set_dirty() {
 #ifdef CONFIG_SHARE
   mstatus->sd = 1;
 #endif
+#ifdef CONFIG_DIFFTEST_REF_SPIKE
```

Comment:
move this ifdef into the above ifdef. They are the same.

Reply:
No, they are different. Actually, "ifdef CONFIG_SHARE" is used when NEMU is used as ref in DiffTest, and "ifdef CONFIG_DIFFTEST_REF_SPIKE" is used when NEMU is used as dut and Spike is used as ref, because Spike update fs and sd in the same time